### DISCUSSION TITLE
Welcome to vscode-dbt-power-user Discussions!

### DISCUSSION THREAD

**Discussion by mdesmet** [AltimateAI/vscode-dbt-power-user#31] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Elektry-On** [comment]
Hi All!
My name is Sven and I am working in Zurich, Switzerland for a Microsoft/Snowflake focused consulting shop.

Wanted to give you a big thank you for your contribution 👏 it really helps to save precious time during development with dbt and it makes it so much easier to handle development related actions (quick compile with shortcut etc.).

Thank you for your hard work!
All the best!
Sven
